source/boundary_temperature/interface.cc

```diff
@@ -96,102 +85,48 @@ namespace aspect
-      // parameters we declare here
-      prm.enter_subsection ("Boundary temperature model");
-      {
-        model_names
-          = Utilities::split_string_list(prm.get("List of model names"));
-
-        AssertThrow(Utilities::has_unique_entries(model_names),
-                    ExcMessage("The list of strings for the parameter "
-                               "'Boundary temperature model/List of model names' contains entries more than once. "
-                               "This is not allowed. Please check your parameter file."));
-
-        const std::string model_name = prm.get ("Model name");
-
-        AssertThrow (model_name == "unspecified" || model_names.size() == 0,
-                     ExcMessage ("The parameter 'Model name' is only used for reasons"
-                                 "of backwards compatibility and can not be used together with "
-                                 "the new functionality 'List of model names'. Please add your "
-                                 "boundary temperature model to the list instead."));
-
-        if (!(model_name == "unspecified"))
-          model_names.push_back(model_name);
-
-        // create operator list
-        std::vector<std::string> model_operator_names =
-          Utilities::possibly_extend_from_1_to_N (Utilities::split_string_list(prm.get("List of model operators")),
-                                                  model_names.size(),
-                                                  "List of model operators");
-        model_operators = Utilities::create_model_operator_list(model_operator_names);
-      }
-      prm.leave_subsection ();
-
-      // go through the list, create objects and let them parse
-      // their own parameters
-      for (unsigned int i=0; i<model_names.size(); ++i)
-        {
-          // create boundary temperature objects
-          boundary_temperature_objects.push_back (std_cxx11::shared_ptr<Interface<dim> >
-                                                  (std_cxx11::get<dim>(registered_plugins)
-                                                   .create_plugin (model_names[i],
-                                                                   "Boundary temperature::Model names")));
-
-          if (SimulatorAccess<dim> *sim = dynamic_cast<SimulatorAccess<dim>*>(boundary_temperature_objects.back().get()))
-            sim->initialize_simulator (this->get_simulator());
-
-          boundary_temperature_objects.back()->parse_parameters (prm);
-          boundary_temperature_objects.back()->initialize ();
-        }
+      return boundary_temperature_objects;
     }
 
 
 
     template <int dim>
     double
-    Manager<dim>::boundary_temperature (const types::boundary_id boundary_indicator,
-                                        const Point<dim> &position) const
+    Manager<dim>::minimal_temperature (const std::set<types::boundary_id> &/*fixed_boundary_ids*/) const
```

Comment:
oh, this negates my comment on the documentation of the function -- the function apparently ignores its argument. is this intended? If so, this is not a virtual function, so it's not possible to overload it, and if the only existing implementation ignores the argument, why not drop it altogether?